core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -401,6 +407,31 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
+  def handleListGroups(): (Errors, List[GroupOverview]) = {
```

Comment:
If the coordinator is still loading some partitions, then it may not be able to return all the groups it actually owns. For example, during a coordinator migration AdminClient's continuous list-group may see some groups temporarily "disappear", how about return the LOADING_IN_PROGRESS error in such cases so that clients can handle it? Again in AdminClients we can then print out some warning message stating some groups are in the progress of migration and hence cannot be displayed.